eq-author/public/index.html

```diff
@@ -8,6 +8,16 @@
     <meta http-equiv="expires" content="0" />
     <meta http-equiv="expires" content="Tue, 01 Jan 1980 1:00:00 GMT" />
     <meta http-equiv="pragma" content="no-cache" />
+    <script>
+      var isChrome =
+        !!window.chrome &&
```

Comment:
Isn't the check for `window.chrome` enough? https://stackoverflow.com/questions/4565112/javascript-how-to-find-out-if-the-user-browser-is-chrome

Reply:
I think it is because some browsers might have the window.chrome variable.

Reply:
Where did you get the check from? The linked stackoverflow seems to suggest `window.chrome` is enough to detect Chromium

Reply:
I got it from: https://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser

The stackoverflow you linked suggest that 

> new Opera 18+ also outputs to true for window.chrome

But I guess it would not be a problem if the page was displayed for an Opera user. So I can remove it if you think it would be better?

Reply:
I guess at the moment we are saying its Chrome only so we should be as restrictive as possible.